.github/workflows/test-runner-on-perf.yml

```
@@ -1,12 +1,90 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Test Runner on Perf Tests
-
+env:
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
+  GOOGLE_CLOUD_PROJECT: broad-jade-perf
+  GOOGLE_CLOUD_DATA_PROJECT: broad-jade-perf-data2
+  TEST_RUNNER_SERVER_SPECIFICATION_FILE: perf.json
+  GOOGLE_ZONE: us-central1
+  K8_CLUSTER: jade-master-us-central1
 on:
   workflow_dispatch: {}
+  #schedule:
+  #  - cron: '0 2 * * *' # run at 2 AM UTC.
 jobs:
-  build:
+  test-runner-perf:
     runs-on: ubuntu-latest
     steps:
-      - name: Run a one-line script
-        run: echo Hello, world!
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+      - name: "Import Vault dev secrets"
+        uses: hashicorp/vault-action@c8b383ee
```

```suggestion
        uses: hashicorp/vault-action@v2.0.1
```

We can probably switch this to `v2.0.1`, it looks like it was finally published.